test/internal/List.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
```

Comment:
This is fine for the time being. I really think a sanctuary-list package makes sense though.